packages/app/src/components/nominationBanner/style.css

```diff
@@ -111,61 +111,60 @@
   overflow: inherit;
   background: #fff;
   box-sizing: border-box;
-  padding: 20px;
   box-shadow: 0 1px 5px rgba(0,0,0,0.7);
   border-radius: 4px;
-  width: 469px;
-  height: 250px;
-  display: flex;
+  width: 800px;
   flex-direction: column;
-  justify-content: space-evenly;
-  align-items: center;
-  align-content: flex-start;
   display: flex;
   pointer-events: visible;
 }
 
 .modal-text {
-  padding: 40px 40px;
-  font-weight: 300;
+  padding: 20px 20px;
+  font-weight: 250;
   letter-spacing: .01em;
 }
 
 .modal-buttons {
   display: flex;
-  justify-content: center;
-  align-items: center;
   line-height: 20px;
-  padding-bottom: 19px;
+  padding-bottom: 3px;
+  padding-left: 580px;
 }
 
 
 .modal-buttons > button {
   color: white;
   border: #999;
   font-weight: bold;
-  padding: 10px 40px 10px 40px;
   margin-right: 10px;
   height: 35px;
-  border-radius: 4rem;
+  border-radius: 1rem;
   font-size: 1rem;
   letter-spacing: .01em;
   line-height: 1rem;
+  text-transform: capitalize;
+  
 }
 
 .button-yes{
   background-color: #128b3e;
 }
+.button-yes:hover{
+  background-color: #128b3ee5;
+}
 
 .button-no{
-  background-color: #f72314;
+  background-color: #999;
+}
+.button-no:hover{
+  background-color: rgba(153, 153, 153, 0.89);
 }
 
 .exit-button {
   cursor: pointer;
   font-size: 30px;
-  padding-left: 475px;
-  padding-top: 5px;
+  padding-left: 770px;
```

Comment:
I think this approach of hardcoding padding sizes to align it to the right may not scale well with different screen sizes. I'd recommend looking into flexbox to align the content to the right instead of using a specific pixel value.

Reply:
looking right now....thanks

Reply:
Done some changes